sql-plugin/src/main/scala/com/nvidia/spark/rapids/RapidsConf.scala

```diff
@@ -1028,6 +1028,10 @@ class RapidsConf(conf: Map[String, String]) extends Logging {
     entry.get(conf)
   }
 
+  def getStringConf(key: String): Option[String] = {
+    conf.get(key)
+  }
```

Comment:
I don't think we want to encourage code looking for confs in RapidsConf that aren't confs explicitly supported by it.  There are parts of the code that explicitly filters the Spark configs to `spark.rapids.*` before passing to code that will build a RapidsConf.

If we want to lookup a SQLConf key then we should use a SQLConf to do so.  `SQLConf.get` can be used to obtain the current active instance.

Reply:
A `RapidsConf` can be built from a `SQLConf` but _this is optional_.  That means one cannot assume that a `RapidsConf` contains `SQLConf` settings, and that's why getting arbitrary keys from a `RapidsConf` should not be supported.